src/main/java/Duke.java

```diff
@@ -1,10 +1,41 @@
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.fxml.FXMLLoader;
+import java.io.IOException;
+import packagedirectory.test.ChatBox;
+import packagedirectory.test.DukeException;
+import packagedirectory.test.Message;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private ChatBox chatBox = new ChatBox("./src/main/duke.txt");
+
+    public String initialize() {
```

Comment:
Try to avoid using relative path as your directory container might change due to jar file.